workflow-testing/src/main/java/com/squareup/workflow1/testing/RealRenderTester.kt

```diff
@@ -344,3 +344,14 @@ internal fun WorkflowIdentifier.realTypeMatchesExpectation(
     }
   }
 }
+
+/**
+ * Falls back to using Java reflection to determine subclass relationship.
+ *
+ * Kotlin's [isSuperclassOf] doesn't play nice with Mockito or Mockk:
+ * `Interface::class.isSuperclassOf(mock<Interface>()::class)` will return false.
+ *
+ * See https://github.com/square/workflow-kotlin/issues/155.
+ */
+private fun KClass<*>.isJavaMockOf(type: KClass<*>): Boolean =
```

Comment:
Any reason not to just use `isAssignableFrom` and drop `isSuperclassOf` entirely?

Reply:
Just keeping the intent clear and highlighting that this is an issue with java mocking libraries. I don't feel too strongly about keeping both checks.

Reply:
I added a link to the Kotlin reflection bug this is working around, would you be ok keeping both now?